packages/common/client-utils/package.json

```diff
@@ -41,11 +38,8 @@
 		"prettier": "prettier --check . --ignore-path ../../../.prettierignore",
 		"prettier:fix": "prettier --write . --ignore-path ../../../.prettierignore",
 		"test": "npm run test:mocha && npm run test:jest",
-		"test:coverage": "npm run test:report",
 		"test:jest": "jest",
-		"test:jest:report": "npm run test:jest -- --ci --coverage --reporters=default --reporters=jest-junit",
```

Comment:
Don't we need these scripts for CI builds in the internal project where code coverage checks are done?

Reply:
For code coverage run, we activate c:test:*:coverage at the root, which call test*:report at the root ,which call test:* (not the test:*:report) in the project.   That is why we don't need these for the pipelines